administrator/modules/mod_submenu/src/Menu/Menu.php

```diff
@@ -207,7 +214,13 @@ public static function preprocess($parent)
                         $iconImage = '<span class="home-image icon-home" aria-hidden="true"></span>';
                         $iconImage .= '<span class="visually-hidden">' . Text::_('JDEFAULT') . '</span>';
                     } elseif (substr($iconImage, 0, 6) === 'image:') {
-                        $iconImage = '&nbsp;<span class="badge bg-secondary">' . substr($iconImage, 6) . '</span>';
+
+                        if ($languageCodeParams) {
```

Comment:
```suggestion
                        if ($languageCodeParams) {
```
Fix PHPCS failing in Drone.